producer/pdu_session.go

```diff
@@ -182,6 +182,12 @@ func HandlePDUSessionSMContextCreate(eventData interface{}) error {
 	establishmentRequest := m.PDUSessionEstablishmentRequest
 	smContext.HandlePDUSessionEstablishmentRequest(establishmentRequest)
 
+	if smContext.SelectedPDUSessionType == nasMessage.PDUSessionTypeUnstructured {
+		smContext.SubPduSessLog.Errorf("Unstructured PDU Session Not Supported")
+		txn.Rsp = smContext.GeneratePDUSessionEstablishmentReject("UnknownPDUSessionType")
```

Comment:
Why does it have to be `UnknownPDUSessionType`? Why not using `PDUSessionTypeIPv4OnlyAllowed`? I am asking this because we do know that the PDU session type is `Unstructured`. However, it is not supported.